### PR TITLE
Fixups

### PR DESCRIPTION
**Pull request by awiouy**
Cosmetic changes

### CODE REVIEW

**Comment by CvH** [comment]
@awiouy maintainer at wrong position :)

**Comment by awiouy** [comment]
fixed, thanks!

File: packages/addons/service/tinc/package.mk [hunk 1]
```diff
@@ -28,14 +28,12 @@ PKG_PRIORITY="optional"
 PKG_SECTION="service/system"
 PKG_SHORTDESC="tinc Virtual Private Network Daemon"
 PKG_LONGDESC="tinc is a virtual private network (VPN) daemon that uses tunnelling and encryption to create a secure private network between hosts on the Internet. Because the VPN appears to the IP level network code as a normal network device, there is no need to adapt any existing software. This allows VPN sites to share information with each other over the Internet without exposing any information to others."
-
 PKG_AUTORECONF="yes"
 
 PKG_IS_ADDON="yes"
 PKG_ADDON_TYPE="xbmc.service"
```
Comment (CvH) [inline packages/addons/service/tinc/package.mk]: `PKG_ADDON_NAME` is missing even if it is redundant (special case here)

File: packages/addons/service/emby/package.mk [hunk 2]
```diff
@@ -28,14 +28,13 @@ PKG_PRIORITY="optional"
...
 
 PKG_IS_ADDON="yes"
 PKG_ADDON_NAME="Emby (beta)"
 PKG_ADDON_TYPE="xbmc.service"
-PKG_ADDON_PROVIDES=""
 PKG_ADDON_REPOVERSION="8.0"
 PKG_ADDON_REQUIRES="tools.mono:0.0.0"
+PKG_MAINTAINER="Anton Voyl (awiouy)"
 
 PKG_AUTORECONF="no"
```
Comment (CvH) [inline packages/addons/service/emby/package.mk]: wrong line :)

**Pull request closed without merging**